packages/launcher/src/utils.ts

```diff
@@ -20,3 +22,52 @@ export function getTreasureLauncherAuthToken(): string | undefined {
 export function isUsingTreasureLauncher(): boolean {
   return !!getTreasureLauncherAuthToken();
 }
+
+export function getTreasureLauncherWalletComponents():
```

Comment:
Looks like a great candidate for a unit test 😏

Reply:
Done, good call